refocus-bdk-server.js

```diff
@@ -28,6 +28,7 @@ const BOTS_ROUTE = '/bots';
 const BOTACTIONS_ROUTE = '/botActions';
 const BOTDATA_ROUTE = '/botData';
 const ROOMS_ROUTE = '/rooms';
+const ROOM_TYPES_ROUTE = '/roomTypes';
```

Comment:
should we just put all these routes ina config file and just add any that are missing that we think are relevant rather than adding 1 by 1